### ISSUE TITLE
Using npx tsconfig.json only opens a file (does not work - Windows)

### ISSUE THREAD

**Issue by max-programming** [benawad/tsconfig.json#43]
Whenever I run `npx tsconfig.json`, it opens a `tsconfig.json` file in `C:\Users\<username>\AppData\Local\npm-cache\_npx\bb22b66c4c94b5c9\node_modules\.bin\tsconfig.json` on Windows. How can I make it work?

![image](https://user-images.githubusercontent.com/51731966/132466709-6a78b376-5249-4378-aa58-8c8673392b3f.png)

**Comment by DannyZolp** [comment]
#18 
the problem resides with windows trying to open the .json file instead of running it in the terminal
(probably) can't be fixed unless ben changed the project name to something else

**Comment by max-programming** [comment]
> #18
> the problem resides with windows trying to open the .json file instead of running it in the terminal
> (probably) can't be fixed unless ben changed the project name to something else

Oh I see. So Powershell is trying to open the file instead of running the script :|

**Comment by davecarrijo** [comment]
So, did you find a way to solve? @max-programming

**Comment by DontEatRice** [comment]
if you have installed git bash for example just type ./node_modules/.bin/tsconfig.json and it will work @Waiolo

**Comment by davecarrijo** [comment]
it just give another error @DontEatRice 
https://i.imgur.com/ABApnY2.png

**Comment by davecarrijo** [comment]
I found out. just  type npx  tsconfig.json.cmd . Just add a cmd at the end.

**Comment by Ross-Ghazi** [comment]
> npx tsconfig.json.cmd

Worked for me.

**Comment by OmgImAlexis** [comment]
Duplicate of https://github.com/benawad/tsconfig.json/issues/47

**Comment by tanishqdaiya** [comment]
None of that worked with me. That's an issue and the ONLY fix I found to be working was to clone the project and build it yourself with a custom name

**Comment by OmgImAlexis** [comment]
@tanishqdaiya did you check the link I provided? If it's not working please show the error, etc. that's happening when you try it.

There should be no reason this doesn't work. https://github.com/benawad/tsconfig.json/issues/47#issuecomment-1061183881

**Comment by tanishqdaiya** [comment]
@OmgImAlexis Thanks for the solution. Using Windows has always been harder for me since I switched from Linux (don't ask me why). Can we set an alias in powershell for that command (`C:\"Program Files"\nodejs\tsconfig.json.cmd`)?

**Comment by OmgImAlexis** [comment]
Yep, add this to your profile. You can find your profile by using this command in powershell `$profile`.

```
function call_tsconfig($args) { C:\"Program Files"\nodejs\tsconfig.json.cmd $args}
Set-Alias tsconfig call_tsconfig
```

Then call it with `tsconfig` in your powershell.

**Comment by DamianArado** [comment]
> I found out. just type npx tsconfig.json.cmd . Just add a cmd at the end.

Thanks, this works! I'm using command prompt on Windows.
